### ISSUE TITLE
Import json to client side ejs template few times (3 ejs templates and 3 jsons data files)

### ISSUE THREAD

**Issue by Scorpovi4** [mde/ejs#431]
This function transform ejs file to html
```
ejs.renderFile(filename, data, options, function(err, str){
    // str => Rendered HTML string
});
```
but only once  (static filename, static data), how could I feed this function 3 ejs files with 3 jsons data for them (couple 1 ejs and 1 data for it) and take 3 rendered html on output the best case with the name of source ejs file?

**Comment by mde** [comment]
This issue has been closed as outdated. 

If you believe this issue is still relevant, please:
1. Open a new issue with current context
2. Reference this issue number (#431)
3. Include details about your use case and EJS version

Thank you for your contribution!
